src/AWS.Deploy.DockerEngine/ProjectUtilities.cs

```diff
@@ -31,9 +31,22 @@ internal static string ReadDockerFileConfig()
         /// <summary>
         /// Reads dockerfile template file
         /// </summary>
-        internal static string ReadTemplate()
+        internal static string ReadTemplate(string? targetFramework)
         {
-            var template = Assembly.GetExecutingAssembly().ReadEmbeddedFile(DockerfileTemplate);
+            string templateLocation;
+            switch (targetFramework)
```

Comment:
I am specifically checking the following target framework to maintain consistency with other parts of the codebase that also have the same check for different purposes.